dmoj/graders/__init__.py

```diff
@@ -1,3 +1,4 @@
 from dmoj.graders.standard import StandardGrader
 from dmoj.graders.signature import SignatureGrader
 from dmoj.graders.custom import CustomGrader
+from dmoj.graders.interactive import InteractiveGrader, Interactor
```

Comment:
I fail to see how anyone would ever benefit from being able to do `from dmoj.graders import Interactor`.